service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/PopulateStandardDirectionsHandlerTest.java

```diff
@@ -101,7 +101,7 @@ void shouldPopulateStandardDirectionsWhenPopulatedDisplayInConfiguration() throw
                 .token(TOKEN)
                 .build());
 
-        given(ordersLookupService.getStandardDirectionOrder()).willReturn(OrderDefinition.builder()
+        given(ordersLookupService.getDirectionOrder()).willReturn(OrderDefinition.builder()
```

Comment:
Why did you rename it?

Reply:
we renamed to reflect reusability. As we needed to use for `CaseManagementOrder` order definitons

Reply:
Reverted change as it isn't actually needed for CMO